### ISSUE TITLE
System Profiling

### ISSUE THREAD

**Issue by hateom** [vslavik/winsparkle#150]
Would it be possible to add System Profiling as it's done in Sparkle?
https://sparkle-project.org/documentation/system-profiling/

**Comment by vslavik** [comment]
Yes, it is *possible*, self-evidently so. All it takes is that *somebody* puts in the work and actually implements it. Sounds like that somebody should be you, given your interest?

**Comment by fagutavikslavik** [comment]
HURR DURR IM VACLAV SLAVIK VERY IMPORTANT MAN I DONT HAVE TIME FOR THIS
